modules/service/src/main/scala/lucuma/odb/service/ExecutionUserCheck.scala

```diff
@@ -17,4 +20,10 @@ trait ExecutionUserCheck {
       case _       => f(user).asLeft
     }
 
+  def checkUser2[F[_]](using Services[F]): Result[Unit] =
+    user.role.access match {
+      case Service => Result.unit // TODO: should specifically be the observe service user, whose name should be passed in the app config
+      case _       => OdbError.NotAuthorized(user.id).asFailure
+    }
+
```

Comment:
This is part of the access control review that I started on but got distracted. It will go away.